### ISSUE TITLE
Add SSL to pages

### ISSUE THREAD

**Issue by Gadflyii** [poolgold/z-nomp-bitcoin-gold#22]
I am trying to add SSL to the z-nomp website, and I get errors trying to start the pool. I am 99.99% sure it is user error. 

On the config.json, under the tls options, are we looking for a path to the certificate and a path to the key file? what format does the cert and the key need to be in? Do you have an example of the tls options in the config.json?

**Comment by martin-key** [comment]
Hello!

I have never used tls in the pool software. I make it though load balancer.

**Comment by Coronon** [comment]
Yes just a path to the files key, and certificate just use letsencrypt
